src/script/view_model/panel/AddParticipantsViewModel.ts

```diff
@@ -62,6 +62,7 @@ export class AddParticipantsViewModel extends BasePanelViewModel {
   selectedService: ko.Observable<ServiceEntity>;
   state: ko.Observable<string>;
   isTeamOnly: ko.PureComputed<boolean>;
+  isServicesEnabled: ko.PureComputed<boolean>;
```

Comment:
Since you only use it in the `showIntegration` computed, you could just make a `const` and not add it to the properties of the class.